mozregression/launchers.py

```diff
@@ -194,6 +194,9 @@ def _install(self, dest):
     def _start(self, profile=None, addons=(), cmdargs=(), preferences=None):
         profile = self._create_profile(profile=profile, addons=addons,
                                        preferences=preferences)
+        if isinstance(profile, FirefoxProfile):
+            profile.preferences['focusmanager.testmode'] = False
+            profile.set_preferences(preferences=profile.preferences)
```

Comment:
Ok, looking closer at this, this seems like a bit of a bandaid and I think there's other preferences in FirefoxProfile (e.g. 'toolkit.telemetry.server') that are not appropriate for mozregression. As a reminder the original FirefoxProfile is defined here: https://dxr.mozilla.org/mozilla-central/source/testing/mozbase/mozprofile/mozprofile/profile.py#345

How about we create a new subclass of Profile (FirefoxRegressionProfile?) for Firefox and set the preferences manually to *just* be things that we need for regression testing (e.g. disabling sending telemetry and health reports). Use your intuition and I'll double check. :)